compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/events/SkikoConverters.kt

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.events
+
+import org.jetbrains.skiko.SkikoInputEvent
+import org.jetbrains.skiko.SkikoInputModifiers
+import org.jetbrains.skiko.SkikoKey
+import org.jetbrains.skiko.SkikoKeyboardEvent
+import org.jetbrains.skiko.SkikoKeyboardEventKind
+import org.jetbrains.skiko.SkikoMouseButtons
+import org.jetbrains.skiko.SkikoPointer
+import org.jetbrains.skiko.SkikoPointerDevice
+import org.jetbrains.skiko.SkikoPointerEvent
+import org.jetbrains.skiko.SkikoPointerEventKind
+import org.jetbrains.skiko.currentNanoTime
+import org.w3c.dom.events.KeyboardEvent
+import org.w3c.dom.events.MouseEvent
+import org.w3c.dom.events.WheelEvent
+import org.w3c.dom.TouchEvent
+import org.w3c.dom.asList
```

Comment:
I took a quick look. I'll review properly on Monday.

Unfortunately, we can't use w3c API from stdlib in common js/wasm source set. These APIs are not common in the stdlibs of js and wasm targets, therefore the metadata compilation will fail when publishing the library. 
It's because js-interop is more limited in k/wasm. 

We can have our internal common subset of w3c API (similar to what was attempted in skiko).

Reply:
sure, this is exactly what I wanted to discuss this Monday, because there's some confusion on my side about this. have a great week-end!